playground/play016.ts

```
@@ -0,0 +1,28 @@
+import { Effect, Data, pipe } from "effect";
+
+class FetchError extends Data.TaggedError("FetchError")<{}> {}
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)